cmake/GetGitRevisionDescription.cmake

```diff
@@ -0,0 +1,168 @@
+# - Returns a version string from Git
```

Comment:
Is it possible to add the original repo from where you got this file? Also are the licensing terms of the original repo/file compatible with moving it into OpenFAST repo?

Reply:
@sayerhs good point. that file is from this widely used github repo: https://github.com/rpavlik/cmake-modules. It is licensed under Boost Version 1.0. The original author and a link to the license are included in the docstring of the files included in our repo so I think we are in the clear legally. For reference, the first paragraph of the license is included below. And I'll add a link to the repo in that file's docstring.

```
Boost Software License - Version 1.0 - August 17th, 2003

Permission is hereby granted, free of charge, to any person or organization
obtaining a copy of the software and accompanying documentation covered by
this license (the "Software") to use, reproduce, display, distribute,
execute, and transmit the Software, and to prepare derivative works of the
Software, and to permit third-parties to whom the Software is furnished to
do so, all subject to the following:
```

Reply:
@rafaelmudafort Let's add a link to the git repo in the CMake file comments. That should suffice.